### PR TITLE
MBS-9923 (I): Convert URL Layout to React

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-9923

Index probably can't be converted until the relationship components are merged, meaning this Layout file is currently unused, but it shouldn't hurt to have it already. Once the relationships component is merged it should be very easy to finish MBS-9923

UrlHeader renamed for consistency on @mwiencek's suggestion.

### CODE REVIEW

File: root/url/UrlHeader.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ type Props = {|
   url: UrlT,
 |};
 
-const URLHeader = ({url, page}: Props) => (
+const UrlHeader = ({url, page}: Props) => (
```
Comment (yvanzo) [inline root/url/UrlHeader.js]: Should ideally be renamed in a separate commit. `root/url/layout.tt` be updated too.

Comment (reosarevok) [reply root/url/UrlHeader.js]: Changed

**Comment by yvanzo** [comment]
Oh OK, so this is dead code... Probably not up for merge right now.
